src/python/py_vlm_pipeline.cpp

```diff
@@ -96,7 +96,7 @@ py::object call_vlm_generate(
 ) {
     auto updated_config = *pyutils::update_config_from_kwargs(generation_config, kwargs);
     ov::genai::StreamerVariant streamer = pyutils::pystreamer_to_streamer(py_streamer);
-
+    py::gil_scoped_release rel;
```

Comment:
I'm not sure whether only `pipe.generate(prompt, images, updated_config, streamer)` should release GIL? what's about `py::cast` ?

CC @as-suvorov do you know?

Reply:
I'd guess this is fine. Even though `py::cast()` returns an object which can be interpreted as a python object, it only belongs to this function until it's returned and not visible to anyone.

Reply:
No, I'm not sure